apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala

```diff
@@ -178,7 +178,7 @@ final case class PathItem(
     patch: Option[Operation] = None,
     trace: Option[Operation] = None,
     servers: List[Server] = List.empty,
-    parameters: List[ReferenceOr[Parameter]] = List.empty,
+    parameters: Set[ReferenceOr[Parameter]] = Set.empty,
```

Comment:
I'm afraid we'll have to de-duplicate in some other way - using a `Set` we might loose the ordering

Reply:
Ok, I was not aware that order of parameters is important here, I will add some de-duplication logic to `EndpointToOpenAPIPaths` then

Reply:
Yes it's better to keep things less surprising ;)